uber/site_sections/emails.py

```diff
@@ -93,6 +93,14 @@ def approve(self, session, ident):
         session.add(ApprovedEmail(ident=ident))
         raise HTTPRedirect('pending?message={}', 'Email approved and will be sent out shortly')
 
+    @csrf_protected
+    def unapprove(self, session, ident):
+        approved_email = session.query(ApprovedEmail).filter(ApprovedEmail.ident == ident).first()
+        if approved_email:
+            session.delete(approved_email)
+            raise HTTPRedirect('pending?message={}', 'Email disapproved and will no longer be sent out')
```

Comment:
Suggest: "Approval to send this e-mail has been rescinded, and no further copies will be sent." or something similar.